service/matching/version_sets.go

```diff
@@ -127,6 +131,13 @@ func UpdateVersionSets(clock hlc.Clock, data *persistence.VersioningData, req *w
 	return data, nil
 }
 
+func hashBuildID(buildID string) string {
+	bytes := []byte(buildID)
+	summed := sha256.Sum256(bytes)
+	// We don't need the padding
+	return strings.ReplaceAll(base64.URLEncoding.EncodeToString(summed[:]), "=", "")
```

Comment:
We mentioned that we might not need all the characters, but that's not a biggie.